objects/obj_p_assra/Step_0.gml

```diff
@@ -41,19 +41,19 @@ if (action="return") and (point_distance(x,y,origin.x,origin.y)<=16){
 
 
 if (action="goto") and (!instance_exists(target)){boarding=false;
-    with(obj_en_ship){if (size=1) then y-=6000;}
+    //with(obj_en_ship){if (size=1) then y-=6000;}
```

Comment:
will you still use these later? if not, then feel free to delete this, we'll have it in git history

Reply:
I will not, it's just an habit I have. Deleted those comments.